src/octree/mod.rs

```diff
@@ -43,12 +43,15 @@ pub use self::factory::OctreeFactory;
 
 mod octree_iterator;
 pub use self::octree_iterator::{
-    contains, intersecting_node_ids, AllPointsIterator, FilteredPointsIterator,
+    contains, intersecting_node_ids, AllPointsIterator, FilteredPointsIterator, NodeIdIterator,
 };
 
 mod batch_iterator;
 pub use self::batch_iterator::{BatchIterator, PointCulling, PointLocation, NUM_POINTS_PER_BATCH};
 
+#[cfg(test)]
```

Comment:
Can we remove these three lines?

Reply:
if it is not here, it does not compile the test

Reply:
Just tried, it also compiles without.

Reply:
resuming from offline talk, it compiles, but the test in octree_test.rs can't be run,